en/micro-integrator/docs/references/mediators/property-Mediator.md

```diff
@@ -77,9 +77,27 @@ by using an XPath function. You can use any of the <a href="../../mediators/prop
 <li><strong>LONG</strong></li>
 <li><strong>SHORT</strong></li>
 <li><p><strong>OM</strong></p></li>
+<li><p><strong>JSON</strong></p></li>
 </ul>
 <p><strong>String</strong> is the default type.</p>
 <p>The <code>OM</code> type is used to set xml property values on the message context. This is useful when the expression associated with the property mediator evaluates to an XML node during mediation. When the <code>OM</code> type is used, the XML is converted to an AXIOM OMElement before it is assigned to a property.</p>
+<p>The <code>JSON</code> type is used to set JSON values on the message context. This comes handy when we use
+ property mediators with JSON payloads as a sub configuration in other native JSON supported mediators (Ex. Enrich
+ ). When we use the Property mediator with <code>JSON</code>, the Enrich mediator switches to JSON processing instead of the
+  default XML processing.  
+  
+  <b>Note</b>: When the JSON is just a string, you need to add quotes around them (due to the restrictions in [RFC](https://tools.ietf.org/html/rfc7159)). 
+  
+  Example 1: Creating a property with a JSON string by giving the value.
```

Comment:
Please build this on MKDocs and see if this builds properly. I don't think this will.
Can't use markdown syntax inside an HTML table.

Reply:
+1 fixed